pulsar-client-cpp/docker/build-wheels.sh

```diff
@@ -26,6 +26,9 @@ BUILD_IMAGE_NAME="${BUILD_IMAGE_NAME:-apachepulsar/pulsar-build}"
 ROOT_DIR=$(dirname $0)/../..
```

Comment:
I think we can change to 
```
ROOT_DIR=`cd $(dirname $0)/../..; pwd`
```
That will be an absolute path.

Reply:
Done